src/bouwdossiers/batch.py

```diff
@@ -90,7 +90,101 @@ def openbaar_to_access(openbaar):
     return models.ACCESS_RESTRICTED
 
 
-def add_dossier(x_dossier, file_path, import_file, count, total_count):  # noqa C901
+def add_wabo_dossier(x_dossier, file_path, import_file, count, total_count):  # noqa C901
+    """
+    Add wabo dossier to the the bouwdossier model. Structure of import is
+    almost identical to pre_wabo xml to avoid confusion since only a few
+    mappings are different and new fields are added.
```

Comment:
Please add a short description of what "wabo" and "pre-wabo" means. That way future readers will have some sort of idea what they're looking at.

Reply:
Good point, I think this is more suited in the README however.